### PR TITLE
Fix: Moving some http links to https

### PR DESCRIPTION
**Pull request by jamesaucode**
> What was wrong/why is this fix needed? (quick summary only)

Found a few http links while I was going through the docs, changing them to https

> MDN URL of main page changed

https://developer.mozilla.org/en-US/docs/Web/API/Console/dirxml
https://developer.mozilla.org/en-US/docs/Web/API/Console/error
https://developer.mozilla.org/en-US/docs/Web/API/Console/group
https://developer.mozilla.org/en-US/docs/Web/API/Console/groupCollapsed
https://developer.mozilla.org/en-US/docs/Web/API/Console/groupEnd
https://developer.mozilla.org/en-US/docs/Web/API/Console/info
https://developer.mozilla.org/en-US/docs/Web/API/Console/log
https://developer.mozilla.org/en-US/docs/Web/API/Console/time
https://developer.mozilla.org/en-US/docs/Web/API/Console/timeLog
https://developer.mozilla.org/en-US/docs/Web/API/Console/trace
https://developer.mozilla.org/en-US/docs/Web/API/Console/warn
https://developer.mozilla.org/en-US/docs/Web/API/Encoding_API
https://developer.mozilla.org/en-US/docs/Web/API/mutationobserver
https://developer.mozilla.org/en-US/docs/Web/API/PaymentAddress
https://developer.mozilla.org/en-US/docs/Web/API/Pointer_events
https://developer.mozilla.org/en-US/docs/Web/API/Touch_events/Using_Touch_Events
https://developer.mozilla.org/en-US/docs/Web/API/Window/name
https://developer.mozilla.org/en-US/docs/Web/CSS
https://developer.mozilla.org/en-US/docs/Web/JavaScript
https://developer.mozilla.org/en-US/docs/Web/Tutorials

> Issue number (if there is an associated issue)

N/A

> Anything else that could help us review it

N/A

### CODE REVIEW

File: files/en-us/web/api/paymentaddress/index.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 <p><span class="seoSummary">The <strong><code>PaymentAddress</code></strong> interface of the <a href="/en-US/docs/Web/API/Payment_Request_API">Payment Request API</a> is used to store shipping or payment address information.</span></p>
 
-<p>It may be useful to refer to the Universal Postal Union web site's <a href="http://www.upu.int/en/activities/addressing/s42-standard.html">Addressing S42 standard</a> materials, which provide information about international standards for postal addresses.</p>
+<p>It may be useful to refer to the Universal Postal Union web site's <a href="https://www.upu.int/en/activities/addressing/s42-standard.html">Addressing S42 standard</a> materials, which provide information about international standards for postal addresses.</p>
```
Comment (wbamberg) [inline files/en-us/web/api/paymentaddress/index.html:17]: This link is 404. Maybe it wants to be https://www.upu.int/en/Postal-Solutions/Programmes-Services/Addressing-Solutions#addressing-s42-standard ?

File: files/en-us/web/api/touch_events/using_touch_events/index.html [hunk 2]
```diff
@@ -131,8 +131,8 @@ <h2 id="Examples_and_demos">Examples and demos</h2>
 <ul>
  <li>{{domxref("Touch_events","Touch Events Overview")}}</li>
  <li><a href="https://developers.google.com/web/fundamentals/design-and-ui/input/touch/touch-events">Implement Custom Gestures</a></li>
- <li><a href="http://www.javascriptkit.com/javatutors/touchevents.shtml">Introduction to Touch events in JavaScript</a></li>
- <li><a href="http://www.codicode.com/art/easy_way_to_add_touch_support_to_your_website.aspx">Add touch screen support to your website (The easy way)</a></li>
+ <li><a href="https://www.javascriptkit.com/javatutors/touchevents.shtml">Introduction to Touch events in JavaScript</a></li>
```
Comment (wbamberg) [inline files/en-us/web/api/touch_events/using_touch_events/index.html:134]: This link gives me a security error, shall we just remove it?

File: files/en-us/web/api/window/name/index.html [hunk 3]
```diff
@@ -32,7 +32,7 @@ <h2 id="Description">Description</h2>
 
 <p><code>Window.name</code> has also been used in some frameworks for providing
   cross-domain messaging (e.g. <a
-    href="http://www.thomasfrank.se/sessionvars.html">SessionVars</a> and Dojo's <a
+    href="https://www.thomasfrank.se/sessionvars.html">SessionVars</a> and Dojo's <a
```
Comment (wbamberg) [inline files/en-us/web/api/window/name/index.html:35]: Let's remove this one.

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thanks for this, @jamesaucode ! I think many of these links are obsolete and should just be removed. I highlighted a few individual ones, but also:

* all the http://www.opera.com/dragonfly/ ones
* http://getfirebug.com/
* http://www.softwareishard.com/

File: files/en-us/web/api/paymentaddress/index.html (hunk 1 above)
Comment (jamesaucode) [reply files/en-us/web/api/paymentaddress/index.html:17]: Yes it seems like the case, will change the link to that instead

**Review by wbamberg — APPROVED** [review]
Thanks for the updates, @jamesaucode !

**Pull request merged**